source/actions.js

```diff
@@ -162,35 +167,48 @@ export const watchEvent = (firebase, dispatch, event, path, dest, onlyLastEvent
   }
 
   const runQuery = (q, e, p) => {
+      dispatch({
+        type: START,
+        timestamp: moment(),
+        requesting : true,
+        requested : false,
```

Comment:
its common pattern on redux to have both requesting and requested, sometimes you want to catch the cases that something was requested before, for example if it was requested don't request it again or for more common case is the difference between loading to no data.